bundles/org.openhab.core.thing/src/main/java/org/eclipse/smarthome/core/thing/binding/ThingHandler.java

```diff
@@ -188,9 +188,13 @@
     void handleRemoval();
 
     /**
-     * This method provides a list of classes which should be registered as services by the framework
+     * This method provides a list of classes which should be registered as services by the framework.
+     * <p>
+     * Use this method for example to return your thing specific discoveries (extending {@link AbstractDiscoveryService}).
+     * For more details have a look at the {@link ThingHandlerService} documentation.
      *
-     * @return - list of classes that will be registered as OSGi services
+     * @return - list of classes that will be registered as OSGi services,
```

Comment:
What's the intention of the dash between `@return` and `list`.
Which documentation style is it?

Reply:
That you have to ask the original contributor I guess

Reply:
Sorry, I did not realize that it has been present before.
But if we identify a bad style (at least we did not use this style if it exists), we can IMHO change it if we already touch the same line.
But absolutely not a must.